app/lib/shared/configuration.dart

```diff
@@ -2,11 +2,17 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:convert' show json;
 import 'dart:io';
 
 import 'package:collection/collection.dart' show UnmodifiableSetView;
+import 'package:equatable/equatable.dart';
```

Comment:
Let's not add this dependency, we can use the serialized JSON string in the test.

Reply:
ok, done :)